client/src/components/CreateQuestion.vue

```diff
@@ -0,0 +1,142 @@
+<template>
+  <div>
+    <div class="container">
+      <h2>Create question</h2>
+      <input class="input-textarea" type="text" placeholder="Question" v-model="text" />
+      <ul id="answers">
```

Comment:
why id?

Reply:
Could be omitted but does not hurt I guess?

Reply:
no it doesn't hurt but doesn't have sense either, I was asking if it actually serves a purpose or you forgot to remove after some change

Reply:
It was just used for styling